dsc/src/resource_command.rs

```diff
@@ -14,14 +15,25 @@ use dsc_lib::{
 };
 use std::process::exit;
 
-pub fn get(dsc: &mut DscManager, resource: &str, input: &Option<String>, stdin: &Option<String>, format: &Option<OutputFormat>) {
+/// Get operation.
+///
+/// # Panics
+///
+/// Will panic if provider-based resource is not found.
```

Comment:
Should never panic as we should exit gracefully

Reply:
Agreed.